android/src/main/kotlin/com/sap/gigya_flutter_plugin/GigyaSDKWrapper.kt

```diff
@@ -224,6 +224,29 @@ class GigyaSDKWrapper<T : GigyaAccount>(application: Application, accountObj: Cl
         })
     }
 
+    /**
+     * Forgot password.
+     */
+    fun forgotPassword(arguments: Any, channelResult: MethodChannel.Result) {
+        val loginId: String? = (arguments as Map<*, *>)["loginId"] as String?
+        if (loginId == null) {
+            currentResult!!.error(MISSING_PARAMETER_ERROR, MISSING_PARAMETER_MESSAGE, mapOf<String, Any>())
+            return
+        }
+        sdk.forgotPassword(loginId, object : GigyaCallback<GigyaApiResponse>() {
+            override fun onSuccess(p0: GigyaApiResponse?) {
+                channelResult.success(null)
```

Comment:
Should not return null here. If there is a response it should be delivered back to the client.

Reply:
Done